aws/resource_aws_ecs_task_definition.go

```diff
@@ -188,6 +188,18 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 
+			"ipc_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
```

Comment:
We can add validation for the `ipc_mode` attribute via:

```go
ValidateFunc: validation.StringInSlice([]string{
  ecs.IpcModeHost,
  ecs.IpcModeNone,
  ecs.IpcModeTask,
}, false),
```

Reply:
Yeah, you are right. I could not find these for some reason.